server/channels/store/sqlstore/system_store.go

```diff
@@ -81,7 +86,8 @@ func (s SqlSystemStore) Get() (model.StringMap, error) {
 
 func (s SqlSystemStore) GetByName(name string) (*model.System, error) {
 	var system model.System
-	if err := s.GetMaster().Get(&system, "SELECT * FROM Systems WHERE Name = ?", name); err != nil {
```

Comment:
Changed this query to read from replica. Is there any reason it should go to master?

Reply:
the reason can be found here https://github.com/mattermost/mattermost/commit/21af1f49f10d7890f3b51d5be99efcbb71c33edd

My suggestion is to keep it as Master